src/main/resources/templates/food_posting.html

```diff
@@ -5,59 +5,168 @@
+    }
+
+    .upload-icon {
+      color: #28a745;
+      font-size: 2rem;
+      margin-bottom: 0.5rem;
+    }
+
+    .upload-text {
+      color: #6c757d;
+    }
+
+    #food-image {
+      display: none;
+    }
+  </style>
 </head>
 <body>
 <div class="main">
   <header th:include="@{/fragments/nav.html}"></header>
 
   <div class="content">
-    <h1>Food Posting Form</h1>
-    <form action="/foodshare/save" method="post" th:object="${foodItem}">
-      <!-- Add input fields -->
-      <div>
-        <label for="foodType">Type of food</label>
-        <input type="text" id="foodType" th:field="*{foodType}" required>
-      </div>
-
-      <div>
-        <label for="quantity">Quantity</label>
-        <input type="number" id="quantity" th:field="*{quantity}" required>
-      </div>
-
-      <div>
-        <label for="allergenInfo">Allergen Information</label>
-        <input type="text" id="allergenInfo" th:field="*{allergenInfo}">
-      </div>
-
-      <div>
-        <label for="pickupTime">Pickup Time</label>
-        <input type="text" id="pickupTime" th:field="*{pickupTime}">
-      </div>
-
-      <div>
-        <label for="location">Location:</label>
-        <input type="text" id="location" th:field="*{location}">
-      </div>
-
-<!--      #TODO add the following fields to the form-->
-<!--      <div>-->
-<!--        <label for="station">Selection from ali stop near by stations</label>-->
-<!--        <input type="text" id="station" th:field="*{station}">-->
-<!--      </div>-->
-
-<!--      <div>-->
-<!--        <label for="food-image">Image:</label>-->
-<!--        <input type="text" id="food-image" th:field="*{food-image}">-->
-<!--      </div>-->
-
-      <button type="submit">Post</button>
-    </form>
-
-    <!--     #TODO pagination -->
+    <div class="food-posting-container">
+      <h1 class="form-title">Food Posting Form</h1>
+
+      <form action="/foodshare/save" method="post" th:object="${foodItem}">
+        <!-- Type of Food -->
+        <div class="form-group">
+          <label for="foodType" class="form-label">Type of food</label>
+          <input type="text" class="form-control" id="foodType" th:field="*{foodType}" required>
+        </div>
+
+        <!-- Quantity -->
+        <div class="form-group">
+          <label for="quantity" class="form-label">Quantity</label>
+          <input type="number" class="form-control" id="quantity" th:field="*{quantity}" required>
+        </div>
+
+        <!-- Allergen Information -->
+        <div class="form-group">
+          <label for="allergenInfo" class="form-label">Allergen information</label>
+          <input type="text" class="form-control" id="allergenInfo" th:field="*{allergenInfo}">
+        </div>
 
+        <!-- Pickup Time -->
+        <div class="form-group">
+          <label for="pickupTime" class="form-label">Pickup time</label>
+          <input type="datetime-local" class="form-control" id="pickupTime" th:field="*{pickupTime}">
```

Comment:
if you change schema, you need to change FoodItem.java.

<img width="607" alt="Screenshot 2025-04-01 at 2 57 01" src="https://github.com/user-attachments/assets/02ddd84a-aa49-4e0c-9aea-edd94afeba3c" />